platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/location/LocationComponentTest.kt

```diff
@@ -97,6 +97,12 @@ class LocationComponentTest {
     Assert.assertEquals(locationEngineRequest, locationComponent.locationEngineRequest)
   }
 
+  @Test
+  fun activateWithDefaultLocationEngineRequestAndOptionsTest() {
+    locationComponent.activateLocationComponent(context, mockk(), true, locationEngineRequest, locationComponentOptions)
+    Assert.assertEquals(locationEngineRequest, locationComponent.locationEngineRequest)
```

Comment:
You could also assert if the `locationComponent#getLocationEngine` is not `null`, and the other way around for a `false` argument.